src/app/app.component.ts

```diff
@@ -11,33 +12,34 @@ import { SidebarService } from '@services/sidebar.service';
 })
 export class AppComponent implements OnInit, OnDestroy {
   @ViewChild('drawer') sidebar: MatDrawer;
-  public isShowModal = false;
-  public isSideNavOpened = false;
-
-  private subModal: Subscription;
-  private subSidenav: Subscription;
+  public isSideNavOpened: boolean = false;
+  public error: string = '';
+  private _subSidenav: Subscription;
 
   constructor(
-    private _modalService: ModalService,
-    private _sidebarService: SidebarService
+    private _sidebarService: SidebarService,
+    private _subjectsService: PostsSubjectsService,
+    private _route: Router
   ) {}
 
-  ngOnInit() {
-    this.isShowModal = this._modalService.isShowModal;
-    this.subModal = this._modalService.isShowModalChanged$.subscribe(
-      (isShowModal) => (this.isShowModal = isShowModal)
-    );
-
-    this.isSideNavOpened = this._sidebarService.isSidenavOpened;
-    this.subSidenav = this._sidebarService.isSidenavChanged$.subscribe(
-      (isSideNavOpened) => {
+  ngOnInit(): void {
+    this._subSidenav = this._sidebarService.isSidenavChanged$.subscribe(
+      (isSideNavOpened: boolean) => {
         this.isSideNavOpened = isSideNavOpened;
       }
     );
+    this._subjectsService._error.subscribe((error: any) => {
```

Comment:
why the error has to be underscored? It is a public property.
Please, do not use underscore notation in Typescript, there are property modifiers